packages/metrics/src/configApi.ts

```diff
@@ -120,11 +121,14 @@ export const fetchLocationChildren = async (id: UUID) => {
 }
 
 export const fetchLocationChildrenIds = async (
-  id: ResourceIdentifier<Location>
+  id: ResourceIdentifier<Location>,
+  typeFilter?: 'CRVS_OFFICE' | 'HEALTH_FACILITY' | 'ADMIN_STRUCTURE'
 ) => {
   // TODO: Migrate InfluxDB to use UUID's instead of the "Location/" prefix
   const locations = await fetchLocationChildren(
     id.replace('Location/', '') as UUID
   )
-  return locations.map(({ id }) => `Location/${id}`)
+  return locations
+    .filter((x) => (typeFilter ? getLocationType(x) === typeFilter : true))
+    .map(({ id }) => `Location/${id}`)
```

Comment:
Just thinking whether it would be worth it to do the filtering via hearth api :thinking:

Reply:
Thanks @Zangetsu101 , agreed.
I've move the filter logic to the mongo query